### ISSUE TITLE
macOS Support

### ISSUE THREAD

**Issue by bamhm182** [bamhm182/FinTrinity#3]
It would be great if there was macOS support. To do this, I need to figure out where QCMA stores its configuration file on macOS. Still trying to figure out how to find it without access to a Mac.

**Comment by hindo** [comment]
Hey. For the MacOS version You can use something like this
```python
def read_qcma_settings():
    filename = os.path.expanduser("~/Library/Preferences/com.codestation.qcma.plist")
    plist = plistlib.loads(open(filename, 'rb').read())
    path = plist['appsPath']
    account = plist['lastAccountId']
    username = plist['lastOnlineId']
```

Hope it helps. I've managed to run `FinTrinity.py` with this.

**Comment by bamhm182** [comment]
You are fantastic! Thank you so much!

On a side note, can you please tell me what this outputs on Mac?

```
import platform
print(platform.system())
```

I expect it to output "Darwin" from my google searches, but I don't have a way to confirm.


EDIT: I just pushed a commit to master with the changes. Would you mind testing it and seeing if it works properly on your machine?

**Comment by bamhm182** [comment]
On a side note, can you please tell me what this outputs on Mac?

```
import platform
print(platform.system())
```

I expect it to output "Darwin" from my google searches, but I don't have a way to confirm.

**Comment by hindo** [comment]
Glad to help.

Regarding the output - here is what I get:
```
>>> import platform
>>> platform.system()
'Darwin'
```

and I'll test as soon as possible the newest changes.

EDIT:

Just tested (twice) the scripts with official Python 3.7.3 installation and it worked but had one issue with
```
psvimg-extract: Permission denied
```
This was an easy fix to do because the downloaded `psvimg` required, as on `Linux` platform, a `chmod` change. I've modified in `decrypt_game` and `encrypt_game` this one line
```
if platform.system() == "Linux" or platform.system() == "Darwin":
```
and it works!

**Comment by bamhm182** [comment]
I was considering putting that in last night, but I opted not to because I thought maybe it would work without it. Thanks for letting me know it doesn't! I added that in and I believe it is working now. If you have a moment, I would really appreciate it if you tested v0.4 on the release page and let me know if it works.

Also, I highly doubt it will work, but can I bother you to see what happens if you download the executable called "FinTrinity-v0.4-Linux" and try running it on your Mac? I imagine it will either not work at all (most likely) or work flawlessly.
